frontend/containers/open-call-page/header/component.tsx

```diff
@@ -60,6 +68,18 @@ export const OpenCallHeader: FC<OpenCallHeaderProps> = ({
     return { consumed, remaining, deadline };
   }, [closing_at, created_at]);
 
+  const handleFavoriteClick = () => {
+    // This mutation uses a 'DELETE' request when the isFavorite is true, and a 'POST' request when is false.
+    favoriteOpenCall.mutate(
+      { id, isFavourite: favourite },
+      {
+        onSuccess: (data) => {
+          openCall.favourite = data.favourite;
+        },
+      }
```

Comment:
I think that more recently, we've stopped doing this and relied on the invalidation provided by React Query instead (see the investor and PD public page). Do you think we could apply the same here?